src/query/frontend/opencypher/grammar/MemgraphCypherLexer.g4

```diff
@@ -93,6 +93,7 @@ SERVICE_URL         : S E R V I C E UNDERSCORE U R L ;
 SESSION             : S E S S I O N ;
 SETTING             : S E T T I N G ;
 SETTINGS            : S E T T I N G S ;
+SHOW_CONFIG         : S H O W UNDERSCORE C O N F I G ;
```

Comment:
Similar to my other comment: as you are using them separately, you have to have the separate words, but `SHOW` and `CONFIG` are already there (CypherLexer.g4 and in this file).